tests/unit-tests/dds/DCPS/.gitignore

```diff
@@ -0,0 +1,2 @@
+/UnitTests_DataSampleHeader
+/SafetyProfilePoolTest
```

Comment:
Not really necessary, but we should probably make the MPC project and executable names consistent when we merge everything together.